lib_eio/generic.mli

```diff
@@ -9,3 +9,22 @@ val probe : #t -> 'a ty -> 'a option
 (** [probe t feature] checks whether [t] supports [feature].
     This is mostly for internal use.
     For example, {!Eio_unix.FD.peek_opt} uses this to get the underlying Unix file descriptor from a flow. *)
+
+(** {2 Closing}
+
+    Resources are usually attached to switches and closed automatically when the switch
+    finishes. However, it can be useful to close them sooner manually in some cases. *)
```

Comment:
```suggestion
    finishes. However, it can be useful to manually close them sooner in some cases. *)
```

Reply:
I got rid of the word `manually` completely. It was awkward in both places.